runtime/mozart/src/xcm_config.rs

```diff
@@ -0,0 +1,289 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! XCM configuration for Mozart.
+
+use super::{
+    parachains_origin, AccountId, AllPalletsWithSystem, Balances, Dmp, Fellows, ParaId, Runtime,
+    RuntimeCall, RuntimeEvent, RuntimeOrigin, TransactionByteFee, Treasury, WeightToFee, XcmPallet,
+};
+
+use crate::governance::StakingAdmin;
+
+use frame_support::{
+    parameter_types,
+    traits::{Contains, Equals, Everything, Nothing},
+    weights::Weight,
+};
+use frame_system::EnsureRoot;
+use mozart_runtime_constants::{currency::CENTS, system_parachain::*};
+use runtime_common::{
+    xcm_sender::{ChildParachainRouter, ExponentialPrice},
+    ToAuthor,
+};
+use sp_core::ConstU32;
+use xcm::latest::prelude::*;
+use xcm_builder::{
+    AccountId32Aliases, AllowExplicitUnpaidExecutionFrom, AllowKnownQueryResponses,
+    AllowSubscriptionsFrom, AllowTopLevelPaidExecutionFrom, ChildParachainAsNative,
+    ChildParachainConvertsVia, DescribeAllTerminal, DescribeFamily, FixedWeightBounds,
+    FrameTransactionalProcessor, FungibleAdapter, HashedDescription, IsChildSystemParachain,
+    IsConcrete, MintLocation, OriginToPluralityVoice, SignedAccountId32AsNative,
+    SignedToAccountId32, SovereignSignedViaLocation, TakeWeightCredit, TrailingSetTopicAsId,
+    UsingComponents, WeightInfoBounds, WithComputedOrigin, WithUniqueTopic,
+    XcmFeeManagerFromComponents, XcmFeeToAccount,
+};
+use xcm_executor::XcmExecutor;
+
+parameter_types! {
+    pub TokenLocation: Location = Here.into_location();
+    pub RootLocation: Location = Location::here();
+    pub const ThisNetwork: NetworkId = NetworkId::Rococo; // TODO(akram): Change to Mozart
```

Comment:
@girazoki do you think this is worth fixing now by using [ByGenesis](https://github.com/moondance-labs/polkadot-sdk/blob/bd3dd974ac6bbb66ca339575edbe590ad9da0afb/polkadot/xcm/src/v4/junction.rs#L128) - if so, what values would be used?

Reply:
for now leave it like this